README.md

```diff
@@ -1,96 +1,117 @@
 # Swift Numerics
   
 ## Introduction
-Swift Numerics provides a set of modules that support numerical computing in Swift.
-These modules fall broadly into two categories:
+
+Swift Numerics provides a set of modules that support numerical computing in Swift. These modules fall broadly into two categories:
```

Comment:
While we don't want to wrap, we _do_ want one sentence per line.
```suggestion
Swift Numerics provides a set of modules that support numerical computing in Swift.
These modules fall broadly into two categories:
```

Reply:
Agreed. The proposed change looks good - will incorporate, thanks @stephentyrone